src/database/models/categories.rs

```diff
@@ -262,6 +263,7 @@ pub struct LoaderBuilder<'a> {
     pub name: Option<&'a str>,
     pub icon: Option<&'a str>,
     pub supported_project_types: Option<&'a [ProjectTypeId]>,
+    pub featured: &'a bool,
```

Comment:
Why are you using a reference here?

Reply:
Dunno, everything else was so I just copied it, removed it now